docs/olm.md

```diff
@@ -8,15 +8,27 @@ Ensure the Hive namespace exists:
 oc new-project hive
 ```
 
-Identify the current version of the operator you wish to replace, if applicable. (TODO)
+Identify the current version of the operator you wish to replace, if applicable. If not, you should be able to just specify a fake version.
```

Comment:
In what format? can you include an example?

Reply:
It really doesn't matter, I'll clarify. I believe the version field is pretty freeform from what I've been told.